packages/web-api/src/methods.ts

```diff
@@ -1305,7 +1306,9 @@ export interface ChatUpdateArguments extends WebAPICallOptions, TokenOverridable
   blocks?: (KnownBlock | Block)[];
   link_names?: boolean;
   parse?: 'full' | 'none';
+  reply_broadcast?: boolean;
```

Comment:
This is included in the public documentation, but didn't yet exist in the chatUpdateArguments.